lms/static/sass/elements/_system-feedback.scss

```diff
@@ -145,6 +145,13 @@
   }
 }
 
+.wrapper-msg {
```

Comment:
To clarify, it seemed like .wrapper-msg-fixed and .wrapper-msg are both the same currently, since they both just extend wrapper-msg. Is that correct?